doc/platform/common_config.md

```diff
@@ -39,30 +42,59 @@ The functional requirements include :
 - Create the common file in the device common directory for different  BRCM switch chip family
 
 - Merge the common config from device common directory to ODM
-platform specific config. Duplicate configuration entries in the platform specific file override entries in the common config.bcm
+platform specific config. Duplicate configuration properties in the platform specific file override the properties in the common config.bcm
+
+Common config. Duplicate configuration properties in the overwrite section of common config override the properties in the platform specific config.bcm ( Feature added in Rev 0.2 )
+
 
 - The final config.bcm merged with common config is required to be copied to a shared folder for debugging   
   
 
 ## 2 Supported Platforms
 
-Per-switching silicon Common config feature is supported on all of the Broadcom platform if the common config files are created in the device/broadcom/x86_64-broadcom_common. Following is the current supported common configuration :
+Per-switching silicon Common config feature is supported on all of the Broadcom platform if the common config files are created in the device/broadcom/x86_64-broadcom_common. Following is the current supported common configuration : In the latest 0.2 updated version, TD4, TH4, and TH5 yml based switch silicons are supported
+in the common config feature.  
 
 ```
+|-- x86_64-broadcom_b27 -- broadcom-sonic-td3.config.bcm
+|-- x86_64-broadcom_b37 -- broadcom-sonic-td3.config.bcm
 |-- x86_64-broadcom_b77 -- broadcom-sonic-td3.config.bcm
+|-- x86_64-broadcom_b78 -- broadcom-sonic-td3.config.bcm
 |-- x86_64-broadcom_b85 -- broadcom-sonic-td2.config.bcm
-|-- x86_64-broadcom_b87-- broadcom-sonic-td3.config.bcm
-|-- x86_64-broadcom_b96-- broadcom-sonic-th.config.bcm
-|-- x86_64-broadcom_b97-- broadcom-sonic-th2.config.bcm
-|-- x86_64-broadcom_b98-- broadcom-sonic-th3.config.bcm
+|-- x86_64-broadcom_b88 -- broadcom-sonic-td4.config.bcm
+|-- x86_64-broadcom_b96 -- broadcom-sonic-th.config.bcm
+|-- x86_64-broadcom_b97 -- broadcom-sonic-th2.config.bcm
+|-- x86_64-broadcom_b98 -- broadcom-sonic-th3.config.bcm
+|-- x86_64-broadcom_b99 -- broadcom-sonic-th4.config.bcm
+|-- x86_64-broadcom_f90 -- broadcom-sonic-th5.config.bcm
```

Comment:
Should TD4, TH4 or TH5's common file be ***.config.**yml** file?

Reply:
It's fine to use the same naming in common file. Only the TH5/TH4/TD4 targeted platform and merged config is yml based. 


Geans

Reply:
ZhaohuiS,

I've committed the changes for HLD review comment. You can check and verify. Fox example as the following,
the properties in the [Overwrite Section] will overwrite the platform config.

[Overwrite Section]
l3_mem_entries=16384
lpm_scaling_enable=0
l3_alpm_enable=0

https://github.com/sonic-net/sonic-sairedis/pull/1411

Geans

Reply:
> ZhaohuiS,
> 
> I've committed the changes for HLD review comment. You can check and verify. Fox example as the following, the properties in the [Overwrite Section] will overwrite the platform config.
> 
> [Overwrite Section] l3_mem_entries=16384 lpm_scaling_enable=0 l3_alpm_enable=0
> 
> [sonic-net/sonic-sairedis#1411](https://github.com/sonic-net/sonic-sairedis/pull/1411)
> 
> Geans

@geans-pin  sorry to miss your latest commit, I have several comments, could you please take a look? thanks

Reply:
All the fixes are committed. Please check.